src/components/Login.js

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import axios from 'axios';
+import { Formik, Form, Field } from 'formik';
+import * as Yup from 'yup';
+import { Container, Header, Button } from 'semantic-ui-react';
+
+const LoginSchema = Yup.object().shape({
+  // telling Formik what shape the input data is supposed to be
+  email: Yup.string()
+    .email('Email is not valid')
+    .required('Please enter your email address'),
+  password: Yup.string()
+    .required('Please enter a password')
+});
+
+function Login (props, { errors, touched, values, isSubmitting }) {
```

Comment:
I don't think you need the second parameter here since those variables handled further below.